app/uk/gov/hmrc/payetaxcalculatorfrontend/quickmodel/TaxResult.scala

```diff
@@ -145,6 +157,14 @@ object TaxResult {
     }
   }
 
+  def moneyFormatter(value: String): String ={
+    val money = """(.*)\.(\d)""".r
```

Comment:
it would be better to change it while serializing from number to string like here:
https://www.safaribooksonline.com/library/view/scala-cookbook/9781449340292/ch02s10.html